src/ResourceManager/Storage/Commands.Management.Storage/Commands.Management.Storage.Netcore.csproj

```diff
@@ -29,10 +29,6 @@
     <WarningsAsErrors />
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="7.1.0-preview" />
```

Comment:
??

Reply:
`<ProjectReference Include="..\..\..\Common\Commands.Common.Storage\Common.Storage.Netcore.csproj" />`
That project holds the common reference to Storage at the moment. It was a later goal to remove that reliance, but for now, it makes sure that anything using it uses the same version of the management library.